arch/arm/soc/atmel_sam/sam4s/soc.c

```diff
@@ -184,7 +172,9 @@ static ALWAYS_INLINE void clock_init(void)
 	while (!(PMC->PMC_SR & PMC_SR_LOCKA))
 		;
 
-	/* Setup PLLB. */
+	/*
+	 * Setup PLLB.
```

Comment:
> According to figure 29-1 (page 516) of the datasheet, it can be used by the Master Clock, the Programmable Clock (an output clock), and the USB Clock.

Good point.